geometry_msgs/Twist velocity_goal
bool drive_auto 
---
geometry_msgs/Twist curr_velocity
---
geometry_msgs/Twist inst_velocity
